scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -418,11 +429,12 @@ def spin_db_update_policy_gate_url(application_name, pipeline_name, json_data):
             return
         
         policy_name = get_policy_name(application_name, pipeline_name, get_stage_name(json_data))
-        output_json['policyName'] = policy_name
-        output_json['policyLink'] = '/policy/' + policy_name
-        json_data['outputs'] = output_json        #
-        logging.info(f"The output after updating policy gate url is: {json_data}")        
-        updated_stage_execution_data(get_stage_id(json_data),json_data)
+        if policy_name is not None:
+           output_json['policyName'] = policy_name
+           output_json['policyLink'] = '/policy/' + policy_name
```

Comment:
hope this will just replace the value if it is already present ?

Reply:
No,  that verification is doing beginning it self